site/src/pages/index.js

```diff
@@ -29,6 +34,11 @@ function Home() {
   return (
     <Layout>
       <Explore />
+
+      <DataPanelWrapper>
+        <DataPanel />
```

Comment:
can we work on naming this better ? 
its a little fuzzy to juse name it `Data` , instead, we can be more semantic about it